modules/@apostrophecms/settings/index.js

```diff
@@ -1,12 +1,53 @@
+const { klona } = require('klona');
+
 module.exports = {
   options: {
     alias: 'settings',
     subforms: {},
-    // XXX not implemented until Phase 2
     groups: {}
   },
 
   init(self) {
+    // List of all allowed protected types and their aliases (`subform.protected: type`).
+    // The key is the type or alias, the value is the actual type (always a string).
+    // All subforms `protected` prop will be converted to the actual type.
+    // Invalid protection types will panic.
+    self.protectedTypes = {
```

Comment:
protectedType => protectionType
protectedTypes => protectionTypes

I found this very confusing at first.